Ai Pin/Stores/ColorStore.swift

```diff
@@ -0,0 +1,27 @@
+import SwiftUI
+
+class ColorStore: ObservableObject {
```

Comment:
Do not use ObservableObject, we use the new @Observable paradigm. This may require you factor things in multiple places.

Reply:
Noted! Changed it to the new @observable paradigm just like the NavigationStore